web/template/index.html

```diff
@@ -12,7 +12,8 @@
         <h1>Todobin</h1>
         <form action="/" method="post">
           <label for="name">List Name</label>
-          <input class="u-full-width" type="text" placeholder="List Name" name="name" id="name" required>
+          <input class="u-full-width" type="text" placeholder="List Name" name="name" id="name" value="{{.Name}}"
+            required>
```

Comment:
Noticed we're not using Name in tplvars... it's not being set anywhere, but I added this before I realized that.